internal/scheduler/kubernetesobjects/affinity/affinity.go

```
@@ -0,0 +1,64 @@
+package affinity
+
+import (
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/armadaproject/armada/internal/common/util"
+)
+
+func AddNodeAntiAffinity(affinity *v1.Affinity, labelName string, labelValue string) {
+	if affinity == nil {
```

why is it ok to silently ignore a nil here?